packages/asset/contracts/RoyaltyHandler.sol

```diff
@@ -0,0 +1,50 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.18;
+
+// initializable
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+abstract contract RoyaltyHandler is Initializable {
```

Comment:
missing natspec contract / functions

Reply:
I was previously only documenting publicly visible functions, but will expand and add dev comments to internal ones as well.